src/components/Data/Documents/DocumentBoxItem.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="DocumentBoxItem card">
+    <div class="card-content">
+      <div class="DocumentBoxItem-title">
+        <span class="card-title">{{document.id}}</span>
+        <dropdown
+          class="DocumentBoxItem-actions"
+          :id="`document-dropdown-${document.id}`"
+          >
+          <li>
+            <a
+              v-bind:class="{'disabled': !canEdit}"
+              @click="editDocument"
+              >
+              Edit
+            </a>
+          </li>
+          <li class="divider"></li>
+          <li>
+            <a
+              v-bind:class="{'disabled': !canDelete}"
```

Comment:
If the user is not allowed to delete the document, he will still see the action button but there will be no effect on click. Maybe it's better to disable it if the user is not allowed to do this.

Despite this design cause an inconsistency between the list and map view and the block view because we are using a button for editing and a dropdown with the delete action for list and map and just a dropdown for block.
[map view action button](https://github.com/kuzzleio/kuzzle-admin-console/pull/444/files#diff-919df2d4cb6f9e0a3a455aace5330b56R18)